### DISCUSSION TITLE
Understanding of the code in order to add a new commerce adapter

### DISCUSSION THREAD

**Discussion by acoronasz** [itswadesh/svelte-commerce#104] [General]
Hi, is it possible to get a 30 mins session to go over the code to understand the following:

1. How to configure to use my pwn Litekart store, I've changed the litekart domain env property but it seems to be loading products from your sample store.
2. What do i need to change in order to use shopify backend
3. What do i need to do to add new commerce adapters 
4. how do themes work

**Comment by itswadesh** [comment]
Hi Alberto,
Sure I can help you understand the code

1. Will help you during the call
2. You need to change the `services` directory for shopify
3. We can discuss about ecommerce adaptors
4. Svelte Commerce does not support theming as we are focusing on clean, light weight and easy to understand code

Regards
Swadesh
Telegram: itswadesh
